test/isolation/specs/dropchunks_race.spec.in

```diff
@@ -0,0 +1,33 @@
+# This file and its contents are licensed under the Apache License 2.0.
+# Please see the included NOTICE for copyright information and
+# LICENSE-APACHE for a copy of the license.
+
+setup {
+  DROP TABLE IF EXISTS dropchunks_race_t1;
+  CREATE TABLE dropchunks_race_t1 (time timestamptz, device int, temp float);
+  SELECT create_hypertable('dropchunks_race_t1', 'time', 'device', 2);
+  INSERT INTO dropchunks_race_t1 VALUES ('2020-01-03 10:30', 1, 32.2);
+
+  CREATE FUNCTION debug_waitpoint_enable(TEXT) RETURNS VOID LANGUAGE C VOLATILE
```

Comment:
Is it possible to separate those functions into some sql util file and import them when they are needed instead of creating each time?

Reply:
Not for isolation tests, unfortunately.